virtual_labs/tests/utils.py

```diff
@@ -243,3 +245,18 @@ async def create_confirmed_setup_intent(customer_id: str) -> SetupIntent:
     )
 
     return intent
+
+
+def wait_until(
```

Comment:
This blocks the event_loop, make this an `Async`function and use asyncio.sleep

Reply:
And done. Thanks @g-bar !